### DISCUSSION TITLE
Add a "Sort Modpacks by Included Mods" or similar option

### DISCUSSION THREAD

**Discussion by TimeBender25** [modrinth/.github#33] [Feature requests]
We've spent more hours of our life than we really should have trying to sift through modpacks looking for ones that include a specific mod or mods that we want to see played with. The recent stitch is trying to find 
More generally, this would be useful for Library Type mods like Valkyrien Skies or Pehkui. But this could also be useful for more Direct Content mods if you want to try out all the different ways people have used mods like Create or Spectrum.

There are two idea we currently have for implementation of such a feature:
1) Add a "Search for packs that include this mod" button to a Mod's own page. This generally lends itself to only looking for a single mod, but would make it easier for users to add a mod to their filter.
2) Add a "**Mods**" Element to the Filters Panel for Modpacks, which could potentially let users select multiple mods to filter the packs for. But requires that the users to know what the mod they're looking for is called.

You wouldn't need to include the filter for each version of the mod, as there's already a Minecraft Version filter that can be used for that.

We're sure it's not as easy as it feels like it should be, as if it were easy it likely would have been done already, but surely as Modpacks are usually just a list of mods already it feels like it should be possible to filter by them.

**Comment by triphora** [comment]
Duplicate of modrinth/labrinth#457
